meson.build

```diff
@@ -0,0 +1,17 @@
+#              Copyright Catch2 Authors
+# Distributed under the Boost Software License, Version 1.0.
+#   (See accompanying file LICENSE_1_0.txt or copy at
+#        https://www.boost.org/LICENSE_1_0.txt)
+
+# SPDX-License-Identifier: BSL-1.0
+
+project(
+  'catch2',
+  'cpp',
+  version : '3.1.0', # CML version placeholder, don't delete
```

Comment:
As an FYI, the "CML" stands for "CMakeList(s)". 😃 
Feel free to leave it in though.

Reply:
Aha! I spent a while searching trying to work out what it stood before and then decided to just copy it exactly. :) Thanks.